theurgy.lic

```diff
@@ -250,10 +270,12 @@ class TheurgyActions
   end
 
   def buy_parchment
-    fput 'read placard'
+    bput('read placard', 'The placard reads')
     pause 1
-    fput 'order 4 from monk'
-    fput 'unroll my parchment'
+    bput('order 4 from monk', 'You decide to purchase the parchment',
```

Comment:
You could return if the order fails (and avoid the 'unroll'). Not a big deal though.